.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,97 @@
+name: Bug
+about: 🐛 Let us know about an unexpected error, a crash, or an incorrect behavior.
+labels:
+  - 'type: bug'
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for opening 🐛 bug report!
+
+- type: textarea
+  attributes:
+    label: A note for the community (please keep)
+    value: |
+      <!-- Please keep this note for the community -->
+      ### Community Note
+      * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+      * If you are interested in working on this issue or have submitted a pull request, please leave a comment
+      <!-- Thank you for keeping this note for the community -->
+
+- type: textarea
+  id: problem
+  attributes:
+    label: Problem
+    description: >
+      Please provide a clear and concise description of what the bug is,
+      including what currently happens and what you expected to happen.
+  validations:
+    required: true
+
+- type: textarea
+  id: config
+  attributes:
+    label: Configuration
+    description: |
+      Paste the relevant parts of your `vector.toml` configuration.
```

```suggestion
      Paste the relevant parts of your Vector configuration file
```
We'll be migrating to YAML at some point in the future

---

Ah, yes, good call 👍

---

81f8d461b